components/customs/SectionHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+
+//interface of expected props
+interface SectionHeaderProps {
+  header_text: string;
+  tooltip_text: string;
+}
+
+const SectionHeader: React.FC<SectionHeaderProps> = ({
+  header_text,
+  tooltip_text,
+}) => {
+  const [showTooltip, setShowTooltip] = useState(false);
+
+  return (
+    <div className="flex items-center space-x-2">
+      <h2 className="text-lg">{header_text}</h2>
+      <div className="relative inline-block">
+        {/* Tooltip */}
+        <TooltipProvider>
+          <Tooltip>
+            {/* Tooltip Trigger */}
+            <TooltipTrigger>
+              <div
+                onMouseEnter={() => setShowTooltip(true)}
+                onMouseLeave={() => setShowTooltip(false)}
+                className="h-5 w-5 text-gray-500 cursor-pointer"
+              >
+                &#9432;
+              </div>
+            </TooltipTrigger>
+            {/* Tooltip Content */}
```

Comment:
The tooltip isn't triggering for me on Firefox or Chrome. I wonder if it works for you if you've done something browser speicific or if one of the merge conflicts somehow broke this functionality?

Reply:
I am using firefox on linux and it works. I think the offset of where to hover is a bit tricky to place. Perhaps can look into making it a click to open instead of a hover.

Reply:
Will test again on chrome to make sure. Not sure at the moment why it would not work.

Reply:
Oh sorry just to clarify my testing was done on a Windows machine.

Reply:
Discussed, I wasn't waiting long enough hahaha